charts/ndb-operator/Chart.yaml

```diff
@@ -23,19 +23,9 @@ icon: https://www.nutanix.com/content/dam/nutanix/global/icons/products/svg/Nuta
 annotations:
   artifacthub.io/changes: |
     - kind: added
```

Comment:
Change this to `security`. Reference thishttps://artifacthub.io/docs/topics/annotations/helm/
```suggestion
    - kind: security
```

Reply:
Ok, changed this one to security and the other to removed.